src/AvtaleSide/steg/KontaktInformasjonSteg/ArbeidsgiverinfoDel/Relasjoner.tsx

```diff
@@ -34,7 +34,8 @@ const Relasjoner: FunctionComponent<Props> = ({ tiltakstype }: Props) => {
     };
 
     const { rolle } = useContext(InnloggetBrukerContext);
-    const [popoverAnker, setPopoverAnker] = useState<HTMLElement | null>(null);
+    const isKanEndreFamilierelasjon =
```

Comment:
Skjønner konvensjonen med `is` for å hinte om at vi snakker om en boolean, men jeg vil jo tenke at `kan` hinter om akkurat det samme?

Reply:
Men `is` er (eller ihvertfall var) en del av java-bean spesifikasjonen en gang i tiden (men det er ikke `kan` 😏). Det var rammeverk som ikke fungerte dersom booleans ikke brukte `get`, `set` eller `is` is for vanlige POJO. Så det er vel bare noe jeg har i fingrene som ikke slipper helt taket 😅